pwa-shell/public/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link href="https://fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
```

Comment:
let's just do the import of the file in the css itself:

`@import url('https://fonts.googleapis.com/css?family=Titillium+Web');`